package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "p2p-graph",
   "description": "Real-time P2P network visualization with D3",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
In the future, can you use the `npm version patch`, `npm version minor`, and `npm version major` commands to change the version? That way, the version changes in a single commit that is easy to see in the commit list. It also adds a git tag called `v1.0.2` that corresponds to the version.